app/src/main/java/de/mathfactory/mooltifill/SettingsActivity.kt

```diff
@@ -154,6 +157,7 @@ class SettingsActivity : AppCompatActivity() {
         supportActionBar?.setHomeButtonEnabled(false)
         permissionSetup(this)
         AwarenessService.ensureService(this)
+        scheduleBlutoothStateObserver()
```

Comment:
Scheduling the observer in SettingsActivity.onCreate() will only get triggered when the user opens the settings app, which might - after reboot - not be the case at all. Also, onDestroy() might be called pretty soon on low-profile devices, unregistering the receiver. Suggestion: Register the receiver in the android manifest with an intent-filter with an `<action android:name="android.bluetooth.device.action.ACL_CONNECTED" />` instead of in code.

Reply:
Done in the local commit.